frontend/src/components/custom-tools/list-of-tools/ListOfTools.jsx

```diff
@@ -133,7 +133,7 @@ function ListOfTools() {
         setListOfTools(tools);
         setAlertDetails({
           type: "success",
-          console: "Deleted successfully",
+          content: "Deleted successfully",
```

Comment:
@tahierhussain Not a blocker, but is it possible to include the deleted item's name here in the error message?

Reply:
Thanks for the catch, @nehabagdia. I have made the change. However, we will have to follow the same practice everywhere. Let's create a separate ticket for this and take it up whenever we have the bandwidth.